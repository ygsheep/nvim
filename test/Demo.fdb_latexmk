# Fdb version 3
["xdvipdfmx"] 1677294213 "Demo.xdv" "Demo.pdf" "Demo" 1677294213
  "Demo.xdv" 1677294213 8244 80526ed9772511ffd541060ab5c47ff2 "xelatex"
  (generated)
  "Demo.pdf"
["xelatex"] 1677294212 "Demo.tex" "Demo.xdv" "Demo" 1677294213
  "/usr/local/share/texmf/tex/xelatex/xecjk/xeCJK-listings.sty" 1677259552 13930 23a8e09aee20f10c713bb1f2be5a2432 ""
  "/usr/local/share/texmf/tex/xelatex/xecjk/xeCJK.cfg" 1677259552 408 7369c27beaa819c5f46de3795b22ee3e ""
  "/usr/local/share/texmf/tex/xelatex/xecjk/xeCJK.sty" 1677259552 183137 ed372550124703b2b73cbcc0b47cb20f ""
  "/usr/local/share/texmf/tex/xelatex/xecjk/xeCJKfntef.sty" 1677259552 39577 b470f8a1dbecc9c45c9bf05ca7db9188 ""
  "/usr/share/texmf-dist/fonts/map/fontname/texfonts.map" 1650183167 3524 cb3e574dea2d1052e39280babc910dc8 ""
  "/usr/share/texmf-dist/fonts/tfm/adobe/zapfding/pzdr.tfm" 1650183167 1528 f853c4d1b4e0550255e02831fdc8496f ""
  "/usr/share/texmf-dist/fonts/tfm/public/cm/cmmi12.tfm" 1650183167 1524 4414a8315f39513458b80dfc63bff03a ""
  "/usr/share/texmf-dist/fonts/tfm/public/cm/cmmi6.tfm" 1650183167 1512 f21f83efb36853c0b70002322c1ab3ad ""
  "/usr/share/texmf-dist/fonts/tfm/public/cm/cmmi8.tfm" 1650183167 1520 eccf95517727cb11801f4f1aee3a21b4 ""
  "/usr/share/texmf-dist/fonts/tfm/public/cm/cmr10.tfm" 1650183167 1296 45809c5a464d5f32c8f98ba97c1bb47f ""
  "/usr/share/texmf-dist/fonts/tfm/public/cm/cmr12.tfm" 1650183167 1288 655e228510b4c2a1abe905c368440826 ""
  "/usr/share/texmf-dist/fonts/tfm/public/cm/cmr6.tfm" 1650183167 1300 b62933e007d01cfd073f79b963c01526 ""
  "/usr/share/texmf-dist/fonts/tfm/public/cm/cmr8.tfm" 1650183167 1292 21c1c5bfeaebccffdb478fd231a0997d ""
  "/usr/share/texmf-dist/fonts/tfm/public/cm/cmsy10.tfm" 1650183167 1124 6c73e740cf17375f03eec0ee63599741 ""
  "/usr/share/texmf-dist/fonts/tfm/public/cm/cmsy6.tfm" 1650183167 1116 933a60c408fc0a863a92debe84b2d294 ""
  "/usr/share/texmf-dist/fonts/tfm/public/cm/cmsy8.tfm" 1650183167 1120 8b7d695260f3cff42e636090a8002094 ""
  "/usr/share/texmf-dist/tex/generic/alphalph/alphalph.sty" 1650181202 11516 2c693377218e7ce5418526bc70496cc1 ""
  "/usr/share/texmf-dist/tex/generic/atbegshi/atbegshi.sty" 1650183167 24708 5584a51a7101caf7e6bbf1fc27d8f7b1 ""
  "/usr/share/texmf-dist/tex/generic/bigintcalc/bigintcalc.sty" 1650183167 40635 c40361e206be584d448876bba8a64a3b ""
  "/usr/share/texmf-dist/tex/generic/bitset/bitset.sty" 1650183167 33961 6b5c75130e435b2bfdb9f480a09a39f9 ""
  "/usr/share/texmf-dist/tex/generic/etexcmds/etexcmds.sty" 1650183167 7734 b98cbb34c81f667027c1e3ebdbfce34b ""
  "/usr/share/texmf-dist/tex/generic/gettitlestring/gettitlestring.sty" 1650183167 8371 9d55b8bd010bc717624922fb3477d92e ""
  "/usr/share/texmf-dist/tex/generic/iftex/iftex.sty" 1650183167 7237 bdd120a32c8fdb4b433cf9ca2e7cd98a ""
  "/usr/share/texmf-dist/tex/generic/infwarerr/infwarerr.sty" 1650183167 8356 7bbb2c2373aa810be568c29e333da8ed ""
  "/usr/share/texmf-dist/tex/generic/intcalc/intcalc.sty" 1650183167 31769 002a487f55041f8e805cfbf6385ffd97 ""
  "/usr/share/texmf-dist/tex/generic/kvdefinekeys/kvdefinekeys.sty" 1650183167 5412 d5a2436094cd7be85769db90f29250a6 ""
  "/usr/share/texmf-dist/tex/generic/kvsetkeys/kvsetkeys.sty" 1650183167 13807 952b0226d4efca026f0e19dd266dcc22 ""
  "/usr/share/texmf-dist/tex/generic/ltxcmds/ltxcmds.sty" 1650183167 17859 4409f8f50cd365c68e684407e5350b1b ""
  "/usr/share/texmf-dist/tex/generic/pdfescape/pdfescape.sty" 1650183167 19007 15924f7228aca6c6d184b115f4baa231 ""
  "/usr/share/texmf-dist/tex/generic/pdftexcmds/pdftexcmds.sty" 1650183167 20089 80423eac55aa175305d35b49e04fe23b ""
  "/usr/share/texmf-dist/tex/generic/stringenc/stringenc.sty" 1650183167 21514 b7557edcee22835ef6b03ede1802dad4 ""
  "/usr/share/texmf-dist/tex/generic/uniquecounter/uniquecounter.sty" 1650183167 7008 f92eaa0a3872ed622bbf538217cd2ab7 ""
  "/usr/share/texmf-dist/tex/latex/atveryend/atveryend.sty" 1650183167 19336 ce7ae9438967282886b3b036cfad1e4d ""
  "/usr/share/texmf-dist/tex/latex/auxhook/auxhook.sty" 1650183167 3935 57aa3c3e203a5c2effb4d2bd2efbc323 ""
  "/usr/share/texmf-dist/tex/latex/base/atbegshi-ltx.sty" 1650183167 3034 3bfb87122e6fa8758225c0dd3cbaceba ""
  "/usr/share/texmf-dist/tex/latex/base/atveryend-ltx.sty" 1650183167 2462 754d6b31b2ab5a09bb72c348ace2ec75 ""
  "/usr/share/texmf-dist/tex/latex/base/fix-cm.sty" 1650183167 14741 04be450f114b49e7fd6f68a0f75f06f0 ""
  "/usr/share/texmf-dist/tex/latex/base/fontenc.sty" 1650183167 4946 461cc78f6f26901410d9f1d725079cc6 ""
  "/usr/share/texmf-dist/tex/latex/base/report.cls" 1650183167 23203 8fbc410e29d3fd675970d5f9698c9c11 ""
  "/usr/share/texmf-dist/tex/latex/base/size12.clo" 1650183167 8449 bc7344e882df4d7e51c046514dee83e4 ""
  "/usr/share/texmf-dist/tex/latex/base/ts1enc.def" 1650183167 8044 fa8875fa7360fa4a5757411aa07dae10 ""
  "/usr/share/texmf-dist/tex/latex/base/tuenc.def" 1650183167 29098 3e2b8f2cbae539bde3ed8f63503c0954 ""
  "/usr/share/texmf-dist/tex/latex/cjk/texinput/CJKfntef.sty" 1650183167 12851 25a9b5cc006609feb5c9a20ce24a5392 ""
  "/usr/share/texmf-dist/tex/latex/ctex/config/ctex-name-utf8.cfg" 1650180511 1769 735a72ed212ebc2ee09cb2f9f3314f5b ""
  "/usr/share/texmf-dist/tex/latex/ctex/config/ctex.cfg" 1650180511 1229 1ecf7cad1a7cea3bfeb01fe4b0543bbb ""
  "/usr/share/texmf-dist/tex/latex/ctex/config/ctexopts.cfg" 1650180511 1290 c880497f2cf4d2bf61a702e2c150e3c0 ""
  "/usr/share/texmf-dist/tex/latex/ctex/ctex.sty" 1650180511 32477 c2d0e742103d29a9ee0eeeccf210e9b6 ""
  "/usr/share/texmf-dist/tex/latex/ctex/ctexhook.sty" 1650180511 6480 4f9af96e535b0eac31a355c667ceb0f2 ""
  "/usr/share/texmf-dist/tex/latex/ctex/ctexpatch.sty" 1650180511 8437 c2b028ffb2ee762e53afc8ee71aee5c2 ""
  "/usr/share/texmf-dist/tex/latex/ctex/engine/ctex-engine-xetex.def" 1650180511 4682 edc388300172b981ed3507192f7c968b ""
  "/usr/share/texmf-dist/tex/latex/ctex/fontset/ctex-fontset-fandol.def" 1650180511 4553 92fee8580533febf6f901e8219ad2e14 ""
  "/usr/share/texmf-dist/tex/latex/ctex/scheme/ctex-scheme-chinese.def" 1650180511 2351 44605f23e4e0e2a54792838449d999c2 ""
  "/usr/share/texmf-dist/tex/latex/float/float.sty" 1650183167 6749 16d2656a1984957e674b149555f1ea1d ""
  "/usr/share/texmf-dist/tex/latex/fontspec/fontspec-xetex.sty" 1650183167 162076 2f6d31c4632f2730c57b9c0fda038e15 ""
  "/usr/share/texmf-dist/tex/latex/fontspec/fontspec.cfg" 1650183167 549 c4adac819276241fea8eb79c5ab7b99e ""
  "/usr/share/texmf-dist/tex/latex/fontspec/fontspec.sty" 1650183167 1656 7e824878bad4df5a3e8bba4e463d9126 ""
  "/usr/share/texmf-dist/tex/latex/graphics-cfg/color.cfg" 1650183167 1213 620bba36b25224fa9b7e1ccb4ecb76fd ""
  "/usr/share/texmf-dist/tex/latex/graphics-cfg/graphics.cfg" 1650183167 1224 978390e9c2234eab29404bc21b268d1e ""
  "/usr/share/texmf-dist/tex/latex/graphics-def/xetex.def" 1650183167 14363 d3c336f2f510c0b73629f44e987d2e6a ""
  "/usr/share/texmf-dist/tex/latex/graphics/color.sty" 1650183167 7197 eb6c1ebf41667a05cb50c23c19d5e8bc ""
  "/usr/share/texmf-dist/tex/latex/graphics/graphics.sty" 1650183167 18399 7e40f80366dffb22c0e7b70517db5cb4 ""
  "/usr/share/texmf-dist/tex/latex/graphics/graphicx.sty" 1650183167 7996 a8fb260d598dcaf305a7ae7b9c3e3229 ""
  "/usr/share/texmf-dist/tex/latex/graphics/keyval.sty" 1650183167 2671 4de6781a30211fe0ea4c672e4a2a8166 ""
  "/usr/share/texmf-dist/tex/latex/graphics/trig.sty" 1650183167 4009 187ea2dc3194cd5a76cd99a8d7a6c4d0 ""
  "/usr/share/texmf-dist/tex/latex/graphviz/graphviz.sty" 1650181372 4062 7b205a4b23203604c3358a8455c55020 ""
  "/usr/share/texmf-dist/tex/latex/hycolor/hycolor.sty" 1650183167 17914 4c28a13fc3d975e6e81c9bea1d697276 ""
  "/usr/share/texmf-dist/tex/latex/hyperref/hxetex.def" 1650183167 45016 0a35faa45750bcb80114f8df2443e9c1 ""
  "/usr/share/texmf-dist/tex/latex/hyperref/hyperref.sty" 1650183167 220999 6145ea83914c186e178d1d31c50b37df ""
  "/usr/share/texmf-dist/tex/latex/hyperref/nameref.sty" 1650183167 13242 133e617c5eebffdd05e421624022b267 ""
  "/usr/share/texmf-dist/tex/latex/hyperref/pd1enc.def" 1650183167 14132 e8e7e61e51ade521a7238fac8362786c ""
  "/usr/share/texmf-dist/tex/latex/hyperref/puenc.def" 1650183167 117004 ed1c2cc82bb9836e9d59549dd8c33098 ""
  "/usr/share/texmf-dist/tex/latex/kvoptions/kvoptions.sty" 1650183167 22521 d2fceb764a442a2001d257ef11db7618 ""
  "/usr/share/texmf-dist/tex/latex/l3backend/l3backend-xetex.def" 1650183167 35763 558b6bb076dfa9b058fe5c58dc6e0434 ""
  "/usr/share/texmf-dist/tex/latex/l3kernel/expl3.sty" 1650183167 6107 e4124ef96db512db87780f95c2a6b136 ""
  "/usr/share/texmf-dist/tex/latex/l3packages/l3keys2e/l3keys2e.sty" 1650183167 4674 ac560df2ef76ab2c406a4948a48f79f8 ""
  "/usr/share/texmf-dist/tex/latex/l3packages/xparse/xparse.sty" 1650183167 6758 7d9d899cbbfc962fbc4bb93f4c69eec2 ""
  "/usr/share/texmf-dist/tex/latex/l3packages/xtemplate/xtemplate.sty" 1650183167 47954 630a20f4097c4a058a3cd46693f91d20 ""
  "/usr/share/texmf-dist/tex/latex/letltxmacro/letltxmacro.sty" 1650183167 5766 13a9e8766c47f30327caf893ece86ac8 ""
  "/usr/share/texmf-dist/tex/latex/listings/listings.cfg" 1650183167 1830 e31effa752c61538383451ae21332364 ""
  "/usr/share/texmf-dist/tex/latex/listings/listings.sty" 1650183167 80964 64e57373f36316e4a09b517cbf1aba2e ""
  "/usr/share/texmf-dist/tex/latex/listings/lstmisc.sty" 1650183167 77022 ee25ce086f4a79d8cf73bac6f94c02a5 ""
  "/usr/share/texmf-dist/tex/latex/refcount/refcount.sty" 1650183167 9878 9e94e8fa600d95f9c7731bb21dfb67a4 ""
  "/usr/share/texmf-dist/tex/latex/rerunfilecheck/rerunfilecheck.sty" 1650183167 9715 b051d5b493d9fe5f4bc251462d039e5f ""
  "/usr/share/texmf-dist/tex/latex/subfigure/subfigure.cfg" 1650181202 2062 a0e7d66e09e508f51289a656aec06ed2 ""
  "/usr/share/texmf-dist/tex/latex/subfigure/subfigure.sty" 1650181202 15188 91281c7ddbccfa54a8e0c3b56ab5aa72 ""
  "/usr/share/texmf-dist/tex/latex/tools/indentfirst.sty" 1650183167 1618 786b14c5a3e144c3a51452d08232d18b ""
  "/usr/share/texmf-dist/tex/latex/url/url.sty" 1650183167 12796 8edb7d69a20b857904dd0ea757c14ec9 ""
  "/usr/share/texmf-dist/tex/latex/zhnumber/zhnumber-utf8.cfg" 1650180511 5646 2609567579096c69f0472004a72489a6 ""
  "/usr/share/texmf-dist/tex/latex/zhnumber/zhnumber.sty" 1650180511 31921 5879df709f54630647c1aa6545bcedb7 ""
  "/usr/share/texmf-dist/web2c/texmf.cnf" 1650183167 39911 2da6c67557ec033436fe5418a70a8a61 ""
  "/var/lib/texmf/web2c/xetex/xelatex.fmt" 1677260927 5825183 a33f57f4e3d937c9bed974514c09b104 ""
  "Demo.aux" 1677294213 921 ac9e27891d4085f9e1ec8793991c286c "xelatex"
  "Demo.out" 1677294213 249 6865ca69d8671f096a056d48069369dd "xelatex"
  "Demo.tex" 1677251978 3135 7cbe41c057d60427ed230ce544f4b204 ""
  "Demo.toc" 1677294213 163 a3fae8330b743ad29d4455f47ee43809 "xelatex"
  (generated)
  "Demo.aux"
  "Demo.log"
  "Demo.out"
  "Demo.toc"
  "Demo.xdv"
