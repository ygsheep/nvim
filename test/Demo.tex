\documentclass[12pt, a4paper]{report}
\usepackage{ctex} % 中文的宏包
\usepackage{indentfirst}
\usepackage{graphicx} % 插入圖片的宏包
\usepackage{float} % 設置圖片浮動位置的宏包
\usepackage{subfigure} % 插入多圖時用子圖顯示宏包
\usepackage{listings} % 代碼塊宏包
\usepackage{color} % 代碼高亮
\usepackage[colorlinks,linkcolor=blue]{hyperref} % URL 包
\usepackage[pdf]{graphviz}
\usepackage{alphalph}
\renewcommand*{\thesubfigure}{(\arabic{subfigure})}

\definecolor{dkgreen}{rgb}{0,0.6,0}
\definecolor{gray}{rgb}{0.5,0.5,0.5}
\definecolor{mauve}{rgb}{0.58,0,0.82}

\lstset{ %
    %language=Octave,                % the language of the code
    basicstyle=\scriptsize\Hack,           % the size of the fonts that are used for the code
    numbers=none,                   % where to put the line-numbers
    numberstyle=\tiny\color{gray},  % the style that is used for the line-numbers
    stepnumber=2,                   % the step between two line-numbers. If it's 1, each line 
                                    % will be numbered
    numbersep=3pt,                  % how far the line-numbers are from the code
    backgroundcolor=\color{white},      % choose the background color. You must add \usepackage{color}
    showspaces=false,               % show spaces adding particular underscores
    showstringspaces=false,         % underline spaces within strings
    showtabs=false,                 % show tabs within strings adding particular underscores
    frame=single,                   % adds a frame around the code
    rulecolor=\color{black},        % if not set, the frame-color may be changed on line-breaks within not-black text (e.g. commens (green here))
    tabsize=2,                      % sets default tabsize to 2 spaces
    captionpos=b,                   % sets the caption-position to bottom
    breaklines=true,                % sets automatic line breaking
    breakatwhitespace=false,        % sets if automatic breaks should only happen at whitespace
    title=\lstname,                   % show the filename of files included with \lstinputlisting;
                                    % also try caption instead of title
    keywordstyle=\color{blue},          % keyword style
    commentstyle=\color{dkgreen},       % comment style
    stringstyle=\color{mauve},         % string literal style
    escapeinside={\%*}{*},            % if you want to add LaTeX within your code
    morekeywords={*,...}               % if you want to add more keywords to the set
}
\setCJKmainfont{Noto Serif CJK TC} % 主要字體 Noto Serif
\newfontfamily\Hack{Hack} % 代碼字體
\author{Huang Po-Hsun}
\date{\today}
\title{數據庫系統 SSD7 實驗報告}
\begin{document}

\maketitle

\tableofcontents

\part{實驗一、數據庫與表的基本操作}

\section{實驗目的}

\begin{itemize}
    \item 熟练掌握一种DBMS的使用方法，完成数据库的创建、删除和连接；数据表的建立、删除；表结构的修改。
    \item 加深对表的实体完整性、参照完整性和用户自定义完整性的理解。
\end{itemize}

\end{document}
