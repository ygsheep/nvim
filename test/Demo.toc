\contentsline {part}{I\hspace {1em}實驗一、數據庫與表的基本操作}{2}{part.1}%
\contentsline {section}{\numberline {0.1}實驗目的}{3}{section.0.1}%
