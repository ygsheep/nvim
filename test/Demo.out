\BOOKMARK [-1][-]{part.1}{\376\377\000I\000\040\133\346\232\127\116\000\060\001\145\170\144\332\136\253\202\007\210\150\166\204\127\372\147\054\144\315\117\134}{}% 1
\BOOKMARK [0][-]{section.0.1}{\376\377\133\346\232\127\166\356\166\204}{part.1}% 2
